### ISSUE TITLE
[6.4.z] Fix all  tests.foreman.ui_airgun.test_contentcredential 

### ISSUE THREAD

**Issue by jameerpathan111** [SatelliteQE/robottelo#6531]
error message:
```
2018-12-04 12:23:30 - widgetastic_null - ERROR - [ContentCredentialEditView/repositories]: An exception happened during read call (elapsed 10017 ms)
2018-12-04 12:23:30 - widgetastic_null - ERROR - [ContentCredentialEditView/repositories]: Message: Could not find an element Locator(by='xpath', locator='.//table')
Traceback (most recent call last):
  File "/home/jenkins/shiningpanda/jobs/a6b158f5/virtualenvs/d41d8cd9/lib/python3.6/site-packages/widgetastic/browser.py", line 335, in element
    return elements[0]
IndexError: list index out of range
```

**Comment by ntkathole** [comment]
@abalakh @oshtaier can you please look into it. This is causing lot of test-failures for 6.4.z and 6.5 as well.

**Comment by oshtaier** [comment]
for 6.5 functionality is completely broken by https://bugzilla.redhat.com/show_bug.cgi?id=1653792

**Comment by oshtaier** [comment]
not sure that 6.4 has any priority for us when we have 95% of failed test cases for 6.5, but maybe you have server with 6.4 or at least screenshot of failure, so we can suggest something

**Comment by ntkathole** [comment]
@oshtaier I looked into the failure. I see the problem while switching tabs in contentcredential view.
Screenshot : [https://imgur.com/1bO9fop](url)
view.read() only returns details tab info (failed to navigate to products and repositories tab). I also tried setting TAB_NAME = 'Products' in nested view but wont help.  Also the locator of product tab is correct i.e view.products.click() works fine. 

```
ipdb> view.sub_widgets                                                                                                                            
[SatVerticalNavigation('.//div[@id="vertical-nav"]/ul'), <airgun.widgets.ContextSelector object at 0x7fc2d22c9470>, SatFlashMessages('//div[@class="toast-notifications-list-pf"]'), <airgun.widgets.ValidationErrors object at 0x7fc2c9052f28>, Text("//a[@id='account_menu']"), <widgetastic_patternfly.BreadCrumb object at 0x7fc2d2319d30>, Text("//button[contains(., 'Remove Content Credential')]"), <airgun.widgets.ConfirmationDialog object at 0x7fc2c87d0518>, <Tab 'Details'>, <Tab 'Products'>, <Tab 'Repositories'>]
```

**Comment by oshtaier** [comment]
@ntkathole to be honest, from your words, it should work just perfect. There is small chance that something wrong with environment, but if both you and @jameerpathan111 can reproduce the problem then only one thing that I can propose is to look on the problem on your server.

Please, send server URL on my email and I will try to find possible root cause of the problem.

**Comment by abalakh** [comment]
I believe it's been addressed in SatelliteQE/airgun#256. widgetastic 0.30.0 (which was used in 6.4.z) had few regressions, which were addressed in 0.30.3. Updating 0.30.0 -> 0.30.3 fixed it for me.
